### ISSUE TITLE
Expose the IP addresses of the device on the website

### ISSUE THREAD

**Issue by pgdr** [FriskByBergen/python-friskby-controlpanel#18]
Put the location of the webserver (address + port) on the front of the controlpanel saying

```
You can reach this site using the address http://10.0.0.2
```

**Comment by pgdr** [comment]
Here is one option:
```python
import socket

def get_socket():
    s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
    s.connect(("friskby.no", 80))
    sockname = s.getsockname()[0]
    s.close()
    return sockname
```

**Comment by pgdr** [comment]
Solved in #22
